src/components/home/Hero/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { Typography } from '@/components/common';
+import { config } from '@/lib';
+import type { PrivateProfile } from '@/lib/types/apiResponses';
+import CheckMark from '@/public/assets/icons/check-mark.svg';
+import LeaderboardIcon from '@/public/assets/icons/leaderboard-icon.svg';
+import ShopIcon from '@/public/assets/icons/shop-icon.svg';
+import Image from 'next/image';
+import Link from 'next/link';
+import { FormEvent, useState } from 'react';
+import styles from './style.module.scss';
+
+interface HeroProps {
+  user: PrivateProfile;
+  checkin: (code: string) => void;
+}
+
+const Hero = ({ user, checkin }: HeroProps) => {
+  const [checkinCode, setCheckinCode] = useState('');
+
+  const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    checkin(checkinCode);
+  };
+
+  return (
+    <div className={styles.hero}>
+      <Image
+        className={`${styles.image} ${styles.desktop}`}
+        src="/assets/graphics/portal/raccoon-waves.svg"
+        alt="Landing page graphic"
+        priority
+        fill
+      />
+      <Image
+        className={`${styles.image} ${styles.mobile}`}
+        src="/assets/graphics/portal/waves.svg"
+        alt="Landing page graphic"
+        priority
+        fill
+      />
+      <div className={styles.content}>
+        <div className={styles.header}>
+          <div>
+            <Typography variant="display/light/large" className={styles.headline}>
+              {'Welcome to ACM, '}
+            </Typography>
+            <Link href={config.profileRoute}>
+              <Typography variant="display/heavy/large" className={styles.headline}>
+                {user.firstName}
+              </Typography>
+            </Link>
+            <Typography variant="display/light/large" className={styles.headline}>
+              !
+            </Typography>
+          </div>
+          <div>
+            <Typography variant="h3/regular" className={styles.inline}>
+              {'You have '}
+            </Typography>
+            <Typography variant="h3/bold" className={styles.inline}>
+              {user.points}
+            </Typography>
+            <Typography variant="h3/regular" className={styles.inline}>
+              {' membership points.'}
+            </Typography>
+          </div>
+        </div>
```

Comment:
is it possible for a user to only have one point? (i.e. is it possible for someone to see "You have 1 membership points"?)

You should use `component="span"` instead of setting `display: inline` on divs. Alternatively, you could reduce code repetition by putting `Link` and `<strong>` inside one `Typography` element, like this:

```suggestion
        <div className={styles.header}>
          <Typography variant="display/light/large" className={styles.headline}>
            Welcome to ACM,&nbsp;
            <Link href={config.profileRoute}>
              <strong>
                {user.firstName}
              </strong>
            </Link>
            !
          </Typography>
          <Typography variant="h3/regular">
            You have&nbsp;
            <strong>
              {user.points}
            </strong>
            &nbsp;membership points.
          </Typography>
        </div>
```

Reply:
I'm going to use component="span" since the text wrapping for the other option was a little off